src/Main.java

```diff
@@ -7,59 +7,65 @@ public static void main(String[] args) {
         // Задача 1
         System.out.println("Задача 1");
         int year = 2025;
-        leapYear(year);
+        if (year < 1584) {
+            System.out.println("Григорианский календарь был введён позже");
+        } else {
+            checkLeapYear(year);
+        }
 
         // Задача 2
         System.out.println();
         System.out.println("Задача 2");
         int clientOS = 0;
         int clientDeviceYear = 2021;
-        selectApp(clientOS, clientDeviceYear);
+        findDeviceApp(clientOS, clientDeviceYear);
 
         // Задача 3
         System.out.println();
         System.out.println("Задача 3");
         int deliveryDistance = 95;
-        if (deliveryDistance > 100) {
+        int days = calculateDeliveryDays(deliveryDistance);
+        if (days > 3) {
             System.out.printf("Доставки нет");
         } else {
-            System.out.println("Для доставки потребуется дней: "+ deliveryTime(deliveryDistance));
+            System.out.println("Для доставки потребуется дней: "+ days);
         }
     }
 
-    public static void leapYear (int year) {
+    public static void checkLeapYear (int year) {
         double year4 = year % 4;
         double year100 = year % 100;
         double year400 = year % 400;
-        if (year <= 1584) {
-            System.out.println("Григорианский календарь был введён позже");
-        } else if ((year4 == 0 && year100 != 0) || year400 == 0) {
-            System.out.println(year + "  год — високосный год");
+        if ((year4 == 0 && year100 != 0) || year400 == 0) {
```

Comment:
```suggestion
        if (year4 == 0 && year100 != 0 || year400 == 0) {
```
скобки не нужны, у И и так приоритет выше, чем у ИЛИ